docs/faq/contributions.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 6
+description: Earning Obol Contributions
+---
+
+# Obol Contributions Program
+
+## What are Contributions?
+
+The Obol Contributions Program acknowledges and rewards participants for their efforts in scaling and decentralizing the Ethereum network by staking on Distributed Validators (DVs). When you stake with DVs, 1% of the (consensus layer) staking rewards are directed to the Obol Collective’s “1% for Decentralization” retroactive funding program (RAF), and the portion of rewards you contribute to the RAF reflects your contributions.
+
+## How to Earn Contributions
+
+You can earn contributions by:
+
+1. **Running your own DV Cluster**: You can create and manage your own DV cluster using our Launchpad. This allows you to directly participate in the network's decentralization efforts and earn contributions. To get started, visit our [DV Launchpad](https://launchpad.obol.org). You can also visit our [Squad Staking](https://squadstaking.com) page for inspiration and to find cluster-mates.
+
+2. **Staking through Staking Partners**: By participating in staking through our partners, your staking rewards will also contribute to the Obol Collective. Here’s how you can start earning contributions through our partners:
+
+- **Lido / Mellow**: The Obol vault accepts ETH deposits into the Simple DVT Module, bridging robust institutional staking and decentralized native staking. [Learn more](https://lido.fi).
```

Comment:
Probably should be a mellow URL or they won't get contributions if they just deposit through the lido front door.

Reply:
Right now, there doesn't seem to be a prod mellow/lido URL available. Once we have it (before the launch), I'll add it

Reply:
likely it'll be https://app.mellow.finance/restake/ethereum-dvsteth, though not yet 100% finalized